### PR TITLE
[bitwarden] Add support for the member data stream

### PR DESCRIPTION
**Pull request by brijesh-elastic**
Type of change
- Enhancement

## What does this PR do?
- Add support for member event. (We've used the /public/members endpoint of https://bitwarden.com/help/api/)

### All changes
- [x] Change follows the [contributing guidelines](https://github.com/elastic/integrations/blob/master/CONTRIBUTING.md)
- [x] Supported versions of the monitoring target is documented
- [x] Supported operating systems are documented (if applicable)
- [x] Integration or [System tests](https://github.com/elastic/elastic-package/blob/master/docs/howto/system_testing.md) exist
- [x] Documentation exists
- [x] Fields follow [ECS](https://github.com/elastic/ecs) and [naming conventions](https://www.elastic.co/guide/en/beats/devguide/master/event-conventions.html)
- [x] At least a manual test with ES / Kibana / Agent has been performed.
- [x] Required Kibana version set to: ^8.4.1

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's changelog.yml file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally

- Clone integrations repo.
- Install elastic package locally.
- Start elastic stack using elastic-package.
- Move to integrations/packages/bitwarden directory.
- Run the following command to run tests.
> elastic-package test 

## Related issue
Closes https://github.com/elastic/integrations/issues/7246

## Automated Test
[test-bitwarden.txt](https://github.com/elastic/integrations/files/13216688/test-bitwarden.txt)

### CODE REVIEW

File: packages/bitwarden/data_stream/member/fields/fields.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+- name: bitwarden
+  type: group
+  fields:
+    - name: member
+      type: group
+      fields:
+        - name: access_all
+          type: boolean
+          description: Determines if this member can access all collections within the organization, or only the associated collections. If set to {true}, this option overrides any collection assignments.
```
Comment (efd6) [inline packages/bitwarden/data_stream/member/fields/fields.yml:9]:
```suggestion
          description: Determines if this member can access all collections within the organization, or only the associated collections. If set to true, this option overrides any collection assignments.
```
or
```suggestion
          description: Determines if this member can access all collections within the organization, or only the associated collections. If set to `true`, this option overrides any collection assignments.
```
Its unclear to me why they used curlies for this in the swagger definition; it's not a parameter and the description is not a parameterisable entity.

File: packages/bitwarden/data_stream/member/fields/fields.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+          fields:
+            - name: name
+              type: keyword
+              description: Organization user status type name.
+            - name: value
+              type: keyword
+              description: Organization user status type value.
+        - name: two_factor_enabled
+          type: boolean
+          description: Returns {true} if the member has a two-step login method enabled on their user account.
```
Comment (efd6) [inline packages/bitwarden/data_stream/member/fields/fields.yml:48]: Same here.

File: packages/bitwarden/data_stream/member/manifest.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+streams:
+  - input: httpjson
+    title: Member logs
+    description: Collect Member logs via API.
+    template_path: httpjson.yml.hbs
+    vars:
+      - name: interval
+        type: text
+        title: Interval
+        description: "Duration between requests to the Bitwarden. NOTE: Supported units for this parameter are h/m/s."
```
Comment (efd6) [inline packages/bitwarden/data_stream/member/manifest.yml:12]:
```suggestion
        description: "Duration between requests to the Bitwarden. Supported units for this parameter are h/m/s."
```

File: packages/bitwarden/data_stream/member/fields/fields.yml (hunk 1 above)
Comment (brijesh-elastic) [reply packages/bitwarden/data_stream/member/fields/fields.yml:9]: Will Update it.

**Review by efd6 — APPROVED** [review]

**Pull request merged**